packages/api/endpoints/collections.js

```diff
@@ -161,5 +180,6 @@ router.put('/:name/:version', put);
 router.delete('/:name/:version', del);
 router.post('/', post);
 router.get('/', list);
+router.get('/granules', granuleList);
```

Comment:
Since we're returning collections and not granules, maybe we should call it `collectionsWithGranules` or `activeCollections`?

Reply:
My thinking was that since the endpoint itself would end up being `/collections/granules`, it would indicate that it's collections with granules. However, now I think you're right, it might imply that it's returning the actual granules rather than the collections themselves. I originally had it as `/active`, so maybe I'll change it back or use one of those suggestions

Reply:
Changed it back to `/active`, added test and updated changelog